examples/common/pigweed/protos/fabric_bridge_service.proto

```diff
@@ -7,6 +7,17 @@ package chip.rpc;
 // Define the message for a synchronized end device with necessary fields
 message SynchronizedDevice {
   uint64 node_id = 1;
+
+  string unique_id = 2;
+  string vendor_name = 3;
+  uint32 vendor_id = 4;
+  string product_name = 5;
+  uint32 product_id = 6;
```

Comment:
Is there a way to say these are optional since these are only needed when calling add, and not needed when calling remove

Reply:
Proto3 should auto-fill them with defaults if they are not sent over the wire. Will add optionality to them to make it clear (I think in that case we will have some has_ items).

Reply:
Updated to optional items in RPC and updated code to take into consideration the `has` values.